index.bs

```diff
@@ -118,14 +114,15 @@ extensible.
 <script>
   const video = document.getElementById('video');
 
-  video.addEventListener('enterpictureinpicture', function(event) {
+  video.addEventListener('enterpictureinpicture', event => {
     // Video entered Picture-in-Picture mode.
     const pipWindow = event.pictureInPictureWindow;
-    console.log('Picture-in-Picture window width: ' + pipWindow.width);
-    console.log('Picture-in-Picture window height: ' + pipWindow.height);
+    const { width, height } = pipWindow;
+    console.log(`Picture-in-Picture window width: ${width}`);
```

Comment:
My personal preference goes with `pipWindow.width` as it is easier to read at first glance.